src/include/btmem.h

```diff
@@ -1071,9 +1071,15 @@ struct __wt_update {
 /*
  * WT_MAX_MODIFY_UPDATE --
  *	Limit update chains to a small value to avoid penalizing reads and
- * permit truncation.
+ *	permit truncation.
  */
-#define	WT_MAX_MODIFY_UPDATE	10
+#define	WT_MAX_MODIFY_UPDATE	10000
```

Comment:
This is no longer a "small value", so if we stick with this we'll need to update the comment.